src/DeserializerFactory.php

```diff
@@ -50,14 +50,23 @@ public function newEntityDeserializer() {
 	}
 
 	/**
-	 * @since 1.1
+	 * @deprecated since 1.4, use newStatementDeserializer instead
```

Comment:
I like the comma much more, because grammar.

I also remove since tags when deprecating stuff. I think the information is not helpful any more when something gets deprecated.